src/CattyTests/DataModel/UserVariableTests.swift

```diff
@@ -70,4 +73,91 @@ final class UserVariableTests: XCTestCase {
         XCTAssertFalse(userVariableA === userVariableCopy)
         XCTAssertTrue(userVariableB === userVariableCopy)
     }
+
+    func testIsEqualToUserVariableForDirectAssignment() {
+        let userVariableA = UserVariable()
+        userVariableA.name = "userVar"
+        userVariableA.value = self.arrayWithSameElement
+
+        var userVariableB = userVariableA
```

Comment:
Can you tell me what this assignment is for? (`var userVariableB = userVariableA`)

Reply:
Hi @m-herold  it was a mistake from my side. I corrected it by using the empty initializer to initialize the `uservariableB`.